librz/core/cgraph.c

```diff
@@ -0,0 +1,384 @@
+// SPDX-FileCopyrightText: 2022 imbillow <billow.fun@gmail.com>
```

Comment:
Have you rewritten *everything* from scratch? If not - please preserve also the original copyrights from `canalysis.c`.